File: sistema veterinario/Anfibio.java
public class Anfibio extends Animal
{
    public Anfibio(String nombre,
                   int edad,
                   double peso,
                   Propietario propietario)
    {
        super(nombre, edad, peso, propietario);
    }

    @Override
    public void aplicarTratamiento()
    {
        System.out.println("Aplicando tratamiento para anfibio.");
    }
}

File: Logica/Anfibio.cs
Anfibio.cs                                                       namespace Sistema_Veterinario.Logica
{
    public class Anfibio : Animal
    {
        public Anfibio(string nombre,
                      int edad,
                      Propietario dueño)
            : base(nombre, edad, dueño)
        {
        }

        public override void AplicarTratamiento()
        {
            Console.WriteLine(
                "Tratamiento para anfibios");
        }
    }
}